wallets/foxwallet-extension/src/config.ts

```diff
@@ -0,0 +1,208 @@
+import { EndpointOptions } from '@cosmos-kit/core';
+
+export const preferredEndpoints: EndpointOptions['endpoints'] = {
+  osmosis: {
+    rpc: ['https://rpc-osmosis.keplr.app/'],
+    rest: ['https://lcd-osmosis.keplr.app/'],
+  },
+  osmosistestnet: {
+    rpc: ['https://rpc-test.osmosis.zone/'],
+    rest: ['https://lcd-test.osmosis.zone/'],
+  },
+  cosmoshub: {
+    rpc: ['https://rpc-cosmoshub.keplr.app'],
+    rest: ['https://lcd-cosmoshub.keplr.app'],
+  },
```

Comment:
hey @ljunb are these your teams' endpoints? The preferred endpoints are meant to be endpoints provided by the wallet. These look like keplr's, do you have a green light from keplr team to use their infra?

Reply:
Hello, thanks for the review. We have confirmed that this configuration is not currently used, so we have deleted it.